modules/javafx.controls/src/test/java/test/javafx/scene/control/ComboBoxTest.java

```diff
@@ -640,7 +640,7 @@ public void selectionModelComboBoxReferenceCanNotBeNull() {
     @Test public void defaultConverterCanHandleIncorrectType_1() {
         ComboBox cb = new ComboBox();
         StringConverter sc = cb.getConverter();
-        assertEquals("42", sc.toString(new Integer(42)));
+        assertEquals("42", sc.toString(Integer.valueOf(42)));
```

Comment:
Autobox?

Reply:
From this test code the type of parameter of `sc.toString()` is not obvious.
`StringConverter` is a template class. `toString()` accepts a template type parameter.
Using autoboxing at such instances would hamper the readability.
So I think autoboxing should be used only when the type is very obvious from just a glance of code.